projects.html

```diff
@@ -0,0 +1,252 @@
+                <h5 class="h3 m-0">
+                  <i
+                    class="fa fa-square m-0"
+                    style="font-size: 1.6rem; color: blue;"
+                  ></i>
+                  MAYA NELSON
+                </h5>
+              </div>
+              <div class="col-12 col-md-auto p-0">
+                <p class="m-0">
+                  <span class="d-none d-md-inline" style="font-size: 1.4rem;">
+                    &nbsp;/
+                  </span>
+                  Project Manager
+                </p>
+              </div>
+            </div>
+          </a>
+
+          <button
+            class="navbar-toggler"
+            type="button"
+            data-bs-toggle="collapse"
+            data-bs-target="#navbarLinks"
+            aria-controls="navbarLinks"
+            aria-label="Toggle navigation"
+          >
+            <i class="fa fa-bars"></i>
+          </button>
+          <div
+            class="justify-content-end p-0 collapse navbar-collapse -success"
+            id="navbarLinks"
+          >
+            <ul class="navbar-nav align-items-end justify-content-end">
+              <li class="nav-item">
+                <a class="nav-link" href="./index.html">ABOUT ME</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="resume.html">RESUME</a>
+              </li>
+
+              <li class="nav-item">
+                <a
+                  class="nav-link"
+                  style="font-weight: bold; color: blue;"
+                  href="#"
+                  >PROJECTS</a
+                >
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="contacts.html">CONTACTS</a>
+              </li>
+            </ul>
+          </div>
+        </nav>
+      </div>
+    </header>
+
+    <main class="container-fluid" style="height: auto;">
+      <div class="main-background row" id="secondDiv"></div>
+      <div id="firstDiv" class="main-content row justify-content-center">
+        <div class="col-12 col-md-7 mb-5 bg-transparent">
+          <h5 class="h3 text-center mb-5">
+            <i
+              class="fa fa-square m-0"
+              style="font-size: 1.2rem; color: blue;"
+            ></i>
+            PROJECTS
+          </h5>
+          <p>
+            I'm a paragraph. Click here to add your own text and edit me. It’s
+            easy. Just click “Edit Text” or double click me to add your own
+            content and make changes to the font. I’m a great place for you to
+            tell a story and let your users know a little more about you.
+          </p>
+        </div>
+        <div class="col-12 col-md-7 mb-5 content-card">
+          <div class="row justify-content-between align-items-center">
+            <div class="col-12 col-md-6">
+              <p class="h3" style="color: blue; font-weight: bold;">
+                Project name 01
+              </p>

```

Comment:
2 problems here:
1) Don't use inline styling, keep your styles in your CSS file. This is both for maintainability and because inline styles 'win' the [specificity battle](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity), which can cause you trouble in the future if you try to override these. 
2) Why choose a `<p>` tag if it could be an `h3` tag? Remember to pick your HTML tags semantically.